data/i18nify-data.md

```diff
@@ -0,0 +1,110 @@
+# Country Attribute Source Data Repository
+
+This repository is dedicated to maintaining version-based Country Attribute Data for SDKs. Our main objectives include:
```

Comment:
dont call it as repository..

What is Country Attribute Data mean? can you add the acronym?